### PR TITLE
Added FR translations

### PR DESCRIPTION
**Pull request by growse**
From #817

### CODE REVIEW

File: project/app/src/main/res/values-fr/strings.xml [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+    <string name="preferencesKeepalive" translatable="false">Keepalive</string>
+    <string name="seconds">Secondes</string>
+    <string name="preferencesProfileId">Mode</string>
+    <string name="mode_mqtt_private_label">MQTT privé</string>
+    <string name="aLocation">Région</string>
+    <string name="events">Événements</string>
+    <string name="title_activity_regions">Régions</string>
+    <string name="title_activity_region">Région</string>
+    <string name="title_activity_map">Map</string>
+    <string name="title_activity_contacts">Copains</string>
```
Comment (jpmens) [inline project/app/src/main/res/values-fr/strings.xml:76]: Copains are more like "Pals". Should we prefer "Amis" as friends?

File: project/app/src/main/res/values-fr/strings.xml [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+    <string name="preferencesUrl" translatable="false">URL</string>
+    <string name="mode_http_private_label">HTTP privé</string>
+    <string name="transitionEntering">entré</string>
+    <string name="transitionLeaving">sorti</string>
+    <string name="errorPreferencesImportFailed">L\'importation a échoué</string>
+    <string name="welcome_play_heading">Services Google Play</string>
+    <string name="welcome_play_description">Owntracks nécessite les services Google Play</string>
+    <string name="icon_description_warning">Attention</string>
+    <string name="welcome_heading">Bienvenu</string>
+    <string name="welcome_description">Les étapes suivantes vous aideront à démarrer avec Owntracks</string>
```
Comment (jpmens) [inline project/app/src/main/res/values-fr/strings.xml:103]: `s/Owntracks/OwnTracks/g`

File: project/app/src/main/res/values-fr/strings.xml [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+    <string name="mode_heading">Mode</string>
+    <string name="mqtt_private_description">Connectez-vous à votre propre configuration Owntracks privée et partagez votre position avec votre famille ou vos amis</string>
+    <string name="permissions_heading">Autorisations</string>
+    <string name="permissions_description">Owntracks nécessite l\'accès à votre position. Veuillez accorder cette autorisation avant de continuer.</string>
+    <string name="permissions_granted">Les autorisations ont été accordées</string>
+    <string name="icon_description_owntracks">Owntracks</string>
+    <string name="enjoy_description">Sélectionnez les préférences dans le menu et configurez OwnTracks pour votre serveur avant de l\'utiliser</string>
+    <string name="done_heading">Done</string>
```
Comment (jpmens) [inline project/app/src/main/res/values-fr/strings.xml:111]: missing. Do we need "terminé"?

File: project/app/src/main/res/values-fr/strings.xml [hunk 4]
```diff
@@ -0,0 +1,164 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="app_name">Owntracks</string>
+    <string name="preferencesServer">"Connection"</string>
+    <string name="preferencesHost" translatable="false">"Host"</string>
+    <string name="preferencesPort" translatable="false">"Port"</string>
+    <string name="preferencesWebsocket">Utiliser WebSockets</string>
+    <string name="preferencesUserUsername">"Nom d'utilisateur"</string>
```
[the remark this answers is not in the document]
Comment (growse) [reply project/app/src/main/res/values-fr/strings.xml:8]: Looks like the surrounding double-quotes get stripped automatically. I'll take them out for consistency's sake though.

File: project/app/src/main/res/values-fr/strings.xml (hunk 2 above)
Comment (growse) [reply project/app/src/main/res/values-fr/strings.xml:103]: Will change this in all the `strings.xml`

File: project/app/src/main/res/values-fr/strings.xml (hunk 3 above)
Comment (growse) [reply project/app/src/main/res/values-fr/strings.xml:111]: Yep - will update.

File: project/app/src/main/res/values-fr/strings.xml (hunk 1 above)
Comment (growse) [reply project/app/src/main/res/values-fr/strings.xml:76]: Agreed

**Pull request merged**